src/controllers/dashboard/plugins/repositories/index.js

```diff
@@ -64,8 +67,7 @@ function getRepositoryElement(repository) {
     const listItem = document.createElement('div');
     listItem.className = 'listItem listItem-border';
 
-    const repoLink = document.createElement('a');
-    repoLink.setAttribute('is', 'emby-linkbutton');
+    const repoLink = createCustomElement('a', 'emby-linkbutton');
     repoLink.className = 'clearLink listItemIconContainer';
```

Comment:
```suggestion
    repoLink.classList.add('clearLink', 'listItemIconContainer');
```

Reply:
Good call. This is exactly what was tripping me up in my testing! :tada: